firmware/user/user_main.c

```diff
@@ -314,6 +344,14 @@ static void ICACHE_FLASH_ATTR pollAccel(void* arg __attribute__((unused)))
         {
             QMA6981_poll(&accel);
         }
+#if SWADGE_VERSION == 1
```

Comment:
Compare against the other #define

Reply:
I think I tried putting SWADGE_BBKIWI (defined in user_config.h) but it didnt seem to be recognized

Reply:
No, the makefile can't pull values from the preprocessor. I'll put a note in the makefile to reference `user_config.h`.